PPH/entities/traits/WhereTrait.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace PPH\entities\traits;
+
+/**
+ * Trait WhereTrait
+ *
+ * @package PPH\entities
+ * @author  Kostas Rentzikas <kostas.rentzikas@gmail.com>
+ */
+trait WhereTrait
+{
+    /** @var string|null */
+    private $andWhereConditions = null;
+
+    /** @var array|null */
+    private $andWhereBindValues = null;
+
+    /** @var array */
+    private static $validOperators = [
+        '=',
+        '>',
+        '<',
+        '>=',
+        '<=',
+        '<>',
+        '!=',
+        '<=>',
+        'LIKE',
+        'NOT LIKE',
+    ];
+
+    /**
+     * @param array $conditions
+     *
+     * @return $this
+     */
+    public function where(array $conditions)
+    {
+        try {
+            $preparedConditions = [];
+            foreach ($conditions as $condition) {
+                if (\count($condition) < 3) {
+                    throw new \Error("Condition must consist of ['column_name', 'operator', 'value']");
+                } elseif ( ! $this->checkValidOperator($condition[1])) {
+                    throw new \Error('Invalid condition operator');
+                } else {
+                    $preparedConditions[] = "{$condition[0]} {$condition[1]}:{$condition[0]}";
```

Comment:
Yep, i should had them prepared too as i did with the values.

Reply:
@renjinsk that won't work because using it as bind param it would just put it around quotes and that will cause a SQL syntax error. A nice approach is to validate if the column exists in the table's schema.